config.json

```diff
@@ -1,17 +1,19 @@
 {
     "FoodOnPlatform": 100,
-    "Team1Agents": 2,
-    "Team2Agents": 2,
-    "Team3Agents": 2,
-    "Team4Agents": 2,
-    "Team5Agents": 2,
-    "Team6Agents": 2,
-    "RandomAgents": 2,
+    "Team1Agents": 0,
+    "Team2Agents": 0,
+    "Team3Agents": 0,
+    "Team4Agents": 0,
+    "Team5Agents": 0,
+    "Team6Agents": 0,
+    "RandomAgents": 0,
+    "Team7Agent1": 10,
```

Comment:
You still haven't changed this back to the original values.

Also please move `RandomAgent` to the last item in the list please.